libs/pika/async_cuda_base/include/pika/async_cuda_base/cublas_handle.hpp

```diff
@@ -27,6 +32,7 @@ namespace pika::cuda::experimental {
         static PIKA_EXPORT cublasHandle_t create_handle(int device, whip::stream_t stream);
 
     public:
+        /// TODO: How to best document constructor and other special member functions.
         PIKA_EXPORT cublas_handle();
```

Comment:
To do. Document that this is default constructible and what the state of a default-constructed handle is.

Reply:
Document this saying it's an invalid handle.